neuralcompression/layers/_hyper_synthesis_transformation_2d.py

```diff
@@ -21,21 +21,47 @@ class HyperSynthesisTransformation2D(Module):
         | https://arxiv.org/abs/1802.01436
 
     Args:
-        in_channels: number of channels in the input signal.
-        features: number of inferred latent features.
+        network_channels: number of channels in the input signal.
+        compression_channels: number of inferred latent features.
     """
 
-    def __init__(self, in_channels: int, features: int):
+    def __init__(
+        self,
+        network_channels: int,
+        compression_channels: int,
+        in_channels: int = 3,
+        activation: Module = ReLU(inplace=True),
```

Comment:
Don't use classes as default arguments. These can create silent bugs. Instead, have the default as `None` and create the class in `__init__`.

Reply:
Done!